packages/sirius-web/frontend/sirius-web-application/src/views/project-browser/create-projects-area/ProjectTemplateCard.tsx

```diff
@@ -159,22 +161,34 @@ export const UploadProjectCard = () => {
   );
 };
 
-export const ShowAllTemplatesCard = ({ onClick }: ShowAllTemplatesCardProps) => {
+export const ShowAllTemplatesCard = () => {
   const classes = useProjectCardStyles();
+  const [state, setState] = useState<CreateProjectAreaState>({
+    page: 0,
+    limit: 3,
+    runningTemplate: null,
+    modalDisplayed: null,
```

Comment:
Don't reuse the state type of another component, 3 out of 4 of those properties make no sense here